pkg/run/install_dashboard.go

```diff
@@ -32,29 +31,33 @@ const (
 	helmRepositoryUrl  = "https://helm.gitops.weave.works"
 )
 
-// InstallDashboard installs the GitOps Dashboard.
-func InstallDashboard(log logger.Logger, ctx context.Context, kubeClient client.Client, kubeConfigArgs *genericclioptions.ConfigFlags, namespace string) error {
-	log.Actionf("Installing the GitOps Dashboard ...")
-
+func GenerateSecret(log logger.Logger) (string, error) {
 	password, err := utils.ReadPasswordFromStdin(log, "Please enter your password to generate the secret: ")
 	if err != nil {
 		log.Failuref("Could not read password")
-		return err
+		return "", err
 	}
 
 	secret, err := bcrypt.GenerateFromPassword([]byte(password), bcrypt.DefaultCost)
 	if err != nil {
 		log.Failuref("Error generating secret from password")
-		return err
+		return "", err
 	}
 
 	log.Successf("Secret has been generated:")
-	fmt.Println(string(secret))
 
+	secretStr := string(secret)
+	fmt.Println(secretStr)
```

Comment:
Could we please stop printing the secret? It doesn't matter too much in practice, but even data derived from passwords should be kept private.

Reply:
Removed printing the secret.